gbpservice/neutron/config/nsf

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
+import sys
+from gbpservice.neutron.nsf.core.main import main
```

Comment:
Rename directories, files, variables from nsf to nfp, network_service to network_function, ns to nf